### PR TITLE
Aluminum Revolution 2: Metallic Boogaloo

### PR DESCRIPTION
**Pull request by Karol1223**
#### Summary
Balance "A few changes, mainly in recipes and disassemblies for aluminum-material items"

#### Purpose of change
The original Aluminum Revolution PR was supposed to set the stage for recipe changes to items with aluminum as their material that utilised steel to craft and here we are now. Thing is there's also a bunch of misc edits connected to aluminum so that's why its the sequel.

#### Describe the solution
The changes are as follows:
- Changed the recipe of various electronics to use scrap aluminum instead of scrap metal (flashlight, radio, two-way radio, hotplate, noise emitter, atomic coffee maker, stun gun, geiger counter, electrohack)
- Changed the hexamine stove recipe to take scrap aluminum instead of small metal sheet
- Changed the back-up beeper recipe to require a plastic chunk
- Changed the tent and large tent recipes to not require screwdriving quality tools
- Changed the aluminum brazing rod recipe to be able to utilise scrap aluminum
- Changed the material of the goo canister to steel (as it uses a steel canister to craft, having it be aluminum makes no sense)
- Slightly increased the volume of scrap aluminum (when initially adding it I messed up the density and scrap had higher density than ingots, this way the density is lower as scrap is supposed to be small assorted pieces as opposed to a full chunk)
- Added the disassembly recipes to a bunch of aluminum items (MRE package, MRE bag, small MRE bag, hand pump, hexamine stove, saddle, compressed air horn, tent, large tent, aluminum tent pole, survival kit box)
- Changed the disassembly recipes for damaged tent and damaged large tent to give less resources (they gave almost all resources required to craft a working un-damaged tent/large tent, not really plausible when the damaged version's masses are halved)
- Removed the duplicate back-up beeper disassembly recipe in the 'uncraft' folder
- Added a new vehicle part: the foldable extra-light frame. Functionally the same as normal extra-light frames but now with extra folding possibilities

#### Describe alternatives you've considered
I considered adding a disassembly recipe for aluminum ingots to take them down scrap, however, a while ago I was told there's someone working on ways to disassemble metal ingots already so I don't want to go into someone's work. I also considered adding the possibility to make MRE bags sealed again but I'm not entirely sure how to go about this so I decided against it.

#### Testing
Booted up the game, didn't throw errors. All edited recipes appear as they should, all items disassemble into what they should. Appears to all work just fine.

### CODE REVIEW

**Comment by liminalRoar** [comment]
Love this. makes me think of one thing that is lacking in Cataclysm: empty soda cans and beer cans. IRL they are very common to find lying beside a road, on the street, in garbage cans, and in excess at any dump.

**Comment by Karol1223** [comment]
> Love this. makes me think of one thing that is lacking in Cataclysm: empty soda cans and beer cans. IRL they are very common to find lying beside a road, on the street, in garbage cans, and in excess at any dump.

I believe they actually can spawn in dumpsters and recycling centers but I do agree they're a bit rare compared to how they are irl. Having said that there's a lot of trash that could just spawn literally on the street, not just the cans. That'd be a fairly large edit though, as far as I can tell, and would most likely require touching mapgen

**Comment by liminalRoar** [comment]
True. Adding it to garbage cans as a common spawn for now wouldn't be too difficult i think. Street side spawns would be neat, but yeah probably a lot of work.

File: data/json/recipes/ammo/weldgas.json [hunk 1]
```diff
@@ -208,6 +208,6 @@
       [ [ "forge", 200 ] ]
     ],
     "proficiencies": [ { "proficiency": "prof_metalworking" } ],
-    "components": [ [ [ "material_aluminium_ingot", 1 ] ] ]
+    "components": [ [ [ "material_aluminium_ingot", 1 ], [ "scrap_aluminum", 15 ] ] ]
```
Comment (Fris0uman) [inline data/json/recipes/ammo/weldgas.json:211]: both amount ot the same mass?

**Review by Fris0uman — COMMENTED** [review]
Can you fix the conflicts?

File: data/json/recipes/ammo/weldgas.json (hunk 1 above)
Comment (Karol1223) [reply data/json/recipes/ammo/weldgas.json:211]: Actually, my bad, they do not. It should be 13 not 15, I'll change this right away

**Comment by Karol1223** [comment]
I believe conflicts should be fixed right now. Now the PR no longer removes scrap aluminum and the goo canister from known_bad list, though, because I no longer have any idea where that'd be

**Pull request merged**